presto-hive/src/main/java/com/facebook/presto/hive/HiveSortOrder.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.hive;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class HiveSortOrder
+{
+    public enum Ordering
```

Comment:
Because we also need to store column names, which I think is beyond what an enum should do. However, now I feel `HiveSortOrder` is kind of confusing. I chose this name because this class is behave like `org.apache.hadoop.hive.metastore.api.Order`, but perhaps we should name this class `SortingColumn` to avoid confusion?